public/js/messages.js

```diff
@@ -119,4 +119,5 @@ $(document).ready(() => {
 // prettier complains that socket is unused.
 // Uncomment this when whoever wants to use it or delete this comment and the following line
 // when it is noticed as an acknowedgement that socket won't be used in this file
-// const socket = io("http://localhost:3000"); << travis barfs on this line
+// eslint-disable-next-line no-unused-vars
```

Comment:
no-unused-vars is disabled?  This is a sleazy cop-out.  Either use the variable or don't have it.  That said, it is better that you disabled it for the next line only rather than the entire file.  When lint errors must be disabled, it is much better to disable it on only that one line than everywhere.  'cause lint is *supposed* to be helpful, right?